### DISCUSSION TITLE
Potential for a Requesting Icons/Sprites Pinned Discussion or something similar

### DISCUSSION THREAD

**Discussion by ThelianTech** [malcolmriley/unused-renders#6] [Ideas and Requests]
So Since Learning about your Idea of wanting to add a way for others to contribute more assets to the repository, I thought it would be a cool Idea to suggest a place for people that find this repository looking for particular things, to potentially have a set discussion or Issue template to be created and submitted for the purpose of requesting different ideas for assets. Since the Idea that you want to do by Restructuring this repository in a way to allow others to contribute their works of art as free assets for the factorio modding community, I think that having a way for a mod developer whos looking for something specific that isnt found here to request such a thing, could be a great feature for other creators/artists to look at and try to come up with something like that idea.

**Comment by malcolmriley** [comment]
Thank you for the suggestion! I agree with what you are saying, that if this project gains other contributors that an issue template or suggestion box or the like should be created for those seeking help with asset creation.

For now, being that it is only me, I don't mind just responding to eMail or discussion posts here or on the Factorio forums. I've renamed this category to "Ideas and Requests" to hopefully make it a bit clearer that making requests is okay (but whether I'd be able to fulfill them is another thing).

Perhaps once these requests start to materialize other contributors will see them and join the effort, but so far not much of either has happened!

**Reply by ThelianTech** [reply]
Yeah, that is true. I would also say theres a good channel in the Factorio Discord that could potentially be useful for spreading word about this repository. I dont know if your in there or not but, if there was any good start to gain popularity it would probably be there. as far as requests. haha. I've got plenty for the mod im making. issue is i wont know fully everything i need til i get closer to an alpha release. Not to mention. i gotta figure out some placeholder graphics i can use similar to those boxes the Factorio devs were using as placeholders for entities missing finalized graphics in all of those FFF's for my own use. but I wouldnt know just yet where to start for that either haha. Its almost like I need a graphics designer to handing all the graphics XD. anyways. Thanks again for the response. and I really hope that this repository will gain enough popularity to be a refuge of common shared factorio assets availible for people to use.

**Reply by malcolmriley** [reply]
I'm not a part of any Factorio Discord servers, but if you feel it would be helpful feel free to post a link to this repository there.

As far as creating your own assets, I have posted a [tutorial section on the repository wiki](https://github.com/malcolmriley/unused-renders/wiki). It might be easier than you think - please take a look and let me know if you have any further questions.
